lms/static/js/verify_student/views/webcam_photo_view.js

```diff
@@ -242,6 +242,11 @@
             // Install event handlers
             $( "#webcam_reset_button", this.el ).on( 'click', _.bind( this.reset, this ) );
             $( "#webcam_capture_button", this.el ).on( 'click', _.bind( this.capture, this ) );
+            $("#webcam_capture_button").keyup(function(event){
+                if(event.keyCode == 13){
+                        $("#webcam_capture_button").click();
+                    }
+                });
```

Comment:
@AlasdairSwan Is this an acceptable pattern for intercepting enter/return key presses?  Or is there a more idiomatic way to do this?